apps/e2ee-server/src/JsBridgeE2EEServer.ts

```
@@ -0,0 +1,138 @@
+    this.socketClient = socketClient;
+    this.setup();
+  }
+
+  private socketClient: Socket;
+
+  override sendAsString = false;
+
+  sendPayload(payload: IJsBridgeMessagePayload | string): void {
+    const p = payload as IJsBridgeMessagePayload;
+    const e = p?.error as { message: string; code: number } | undefined;
+    if (e && e?.code && e?.code === CLIENT_TO_CLIENT_RATE_LIMIT_ERROR_CODE) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)